src/cache/index.js

```diff
@@ -15,3 +15,4 @@
  */
 
 export * from './actions';
+export { default as SimpleCache } from './SimpleCache';
```

Comment:
I think that we may be able to add ImmutableCache here if tree shaking will stop the attempted import of immutable

Reply:
This doesnt work currently without requiring `Immutable` to be installed in practice.

Reply:
Best, for deferring to Rollup Yarn Multirepo effort where we separate these out into packages for simplicity sake.